msprime/simulations.py

```diff
@@ -1929,12 +1929,49 @@ def get_ll_representation(self):
 
 
 class BetaCoalescent(ParametricSimulationModel):
-    # TODO document.
+    """
+    A diploid Xi-coalescent with up to four simultaneous multiple mergers and
+    crossover recombination. The string ``"beta"`` can be used to refer to this
+    model.
+
+    There are two main differences between the Beta-Xi-coalescent and the
+    standard coalescent. Firstly, the number of lineages that take part in each
+    common ancestor event is random, with distribution determined by moments of
+    the Beta(`2 - alpha`, `alpha`)-distribution. All participating lineages are
```

Comment:
Maybe use ``:math:`` here and elsewhere in these descriptions? It works pretty well (see the "simulation model" section for examples of it being used.) https://www.sphinx-doc.org/en/1.0/ext/math.html